xline/src/server/command.rs

```diff
@@ -215,20 +216,27 @@ where
     index_barrier: Arc<IndexBarrier>,
     /// Barrier for propose id
     id_barrier: Arc<IdBarrier>,
+    /// Revision Number generator for KV request and Lease request
+    general_rev: Arc<RevisionNumber>,
+    /// Revision Number generator for Auth request
+    auth_rev: Arc<RevisionNumber>,
 }
 
 impl<S> CommandExecutor<S>
 where
     S: StorageApi,
 {
     /// New `CommandExecutor`
+    #[allow(clippy::too_many_arguments)]
     pub(crate) fn new(
         kv_storage: Arc<KvStore<S>>,
         auth_storage: Arc<AuthStore<S>>,
         lease_storage: Arc<LeaseStore<S>>,
         persistent: Arc<S>,
         index_barrier: Arc<IndexBarrier>,
         id_barrier: Arc<IdBarrier>,
+        general_rev: Arc<RevisionNumber>,
+        auth_rev: Arc<RevisionNumber>,
```

Comment:
Should we put these changes in this PR? This PR is about adding a prepare phase, right?

Reply:
Yes, adding a `prepare` phase to a command is not just a matter of defining the relevant method in the `CommandExecutor` and the `Command` trait, it's also a matter of implementing the corresponding responsibilities. Therefore, we need to move the revision generation logic into the "prepare" phase in this pr.